packages/storybook/src/documentation/pages/sorting.stories.mdx

```diff
@@ -114,18 +116,64 @@ const columns: Columns = [
 
 <SortIndexDemo />
 
+### Using the `sortModel` options
+
+XGrid also offers to override the sortModel using its `sortModel` property of its `options` React prop.
+**Note** The `sortModel` property is an array and its order will be used as the `sortIndex` property described above.
+
+```tsx
+<XGrid
+  rows={rows}
+  columns={columns}
+  options={{
+    sortModel = [
+      { field: 'age', sortDirection: 'desc' },
+      { field: 'name', sortDirection: 'asc' },
+    ],
+  }}
+/>
+```
+
+<SortModelOptionDemo />
+
+## Server-side sorting
+
+XGrid supports both client and server-side pagination. By default, sorting works on the client-side. To switch it to server-side,
+set the property `sortingMode` to `server` as string or you can use our `FeatureModeConstant`.
+
+Then, you're going to need to handle the `onColumnsSortingChange` event, to intercept the event, sort the rows on the server side and update the component with the new sorted rows.
+
+```tsx
+const [rows, setRows] = React.useState<RowsProp>([]);
+const onColumnsSortingChange = (params) => {
+  sortServerRows(params).then((newRows) => {
+    setRows(newRows);
```

Comment:
Unsafe

Reply:
Let's treat this one separately.
Should we manage concurrency in demos? maybe an RFC, or doc feedback
There is no server here so I don't think there is any point... 

Maybe we should do a demo with a real api? 🤔

Reply:
> concurrency in demos?

I don't think that this important. I think that what matter is either or not developers can trust our demos, to behave bug free.

Reply:
it will be as we have a fixed timeout so, latest result will always be the correct one